metricbeat/files.txt

```diff
@@ -0,0 +1 @@
+documents.json
```

Comment:
This file is needed for the script that prepares the corpus for offline usage (see https://github.com/elastic/rally-tracks/blob/master/download.sh). I think when you're done this should only contain the compressed corpus file (+ a smaller file that contains the test corpus, see our [docs](https://esrally.readthedocs.io/en/stable/adding_tracks.html#adding-support-for-test-mode)).

Reply:
I think this should contain:

```
documents.json.bz2
documents-1k.json.bz2
```

For details please see my comment above.